.kitchen.yml

```diff
@@ -1,78 +1,37 @@
 ---
 driver:
-  name: dokken
+  name: vagrant
```

Comment:
I propose to add multiple kitchen files as we are doing in https://github.com/chef/inspec 
Reason: we cannot run vagrant in travis. Still like to see the vagrant option within the cookbook. Just move it to .kitchen.vagrant.yml